File: api/code-samples/python/personal/lockAccount.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"

sdk = OpenSDK(host)
personal_response = sdk.personal.lock_account(address)

print(json.loads(personal_response.response.data))

File: web3rpc/rpc-specs/api/code-samples/javascript/personal/lockAccount.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const address = "0xfa415bb3e6231f488ff39eb2897db0ef3636dd32"

    sdk.personal.lockAccount(address,{}, (err, data, response) => {
        console.log(data);
    });
}
)()